.changeset/large-socks-live.md

```diff
@@ -0,0 +1,8 @@
+---
+'@remirror/extension-react-tables': patch
+'@remirror/react-hooks': patch
```

Comment:
@ocavue should these be major? I wanted to avoid accidentally bumping to 3.0

Reply:
I think this should be major. Changeset will submit a pull request, which includes the update for the `version` field in `package.json`. So we should be able to review the version number and avoid unexpected 3.0 before the package is published to NPM.